config/datasets/gcp.yaml

```diff
@@ -38,9 +43,13 @@ catalog_study_index: ${datasets.outputs}/catalog_study_index
 catalog_study_locus: ${datasets.outputs}/catalog_study_locus
 finngen_study_index: ${datasets.outputs}/finngen_study_index
 ukbiobank_study_index: ${datasets.outputs}/ukbiobank_study_index
+ukbiobank_sumstats: ${datasets.outputs}/ukbiobank_sumstats
+ukbiobank_sumstats_merged: ${datasets.outputs}/ukbiobank_sumstats_merged
 
 # Templates
 ld_index_template: ${datasets.outputs}/ld_indices/gnomad_r2.1.1.{POP}.common.ld.variant_indices
 finngen_release_prefix: FINNGEN_R9
 finngen_sumstat_url_prefix: https://storage.googleapis.com/finngen-public-data-r9/summary_stats/finngen_R9_
 finngen_sumstat_url_suffix: .gz
+ukbiobank_study_id: NEALE2_100001_raw
```

Comment:
I had ```NEALE2_100001_raw``` for testing as I used ```ukbiobank_study_id``` in the code similar to how ```study_id``` is used in the gwascatalog code. However, since I merge all the files and create the studyId column at the point of merging potentially I don't need ```ukbiobank_study_id``` at all since it's already in place

Reply:
Got it, so then this variable will not be present in the final config version — let's keep this thread open until then